policy/policy_test.go

```diff
@@ -333,7 +333,7 @@ func TestGetEnabled(t *testing.T) {
 			name:                  "request types limit enabled checks",
 			argsChecks:            []string{},
 			requiredRequestTypes:  []checker.RequestType{checker.FileBased, checker.CommitBased},
-			expectedEnabledChecks: 5, // All checks which are FileBased and CommitBased
+			expectedEnabledChecks: 7, // All checks which are FileBased and CommitBased
```

Comment:
hmm, i thought this PR added FileBased to 3 checks that weren't before? Any idea what's going on here?

Reply:
I think that test is for FileBased **and** Commit Based. I added 3 checks that were FileBased but two originally had CommitBased.